dfs/registry/index.go

```diff
@@ -96,11 +98,19 @@ func (client *RegistryIndexClient) PushImage(image, uuid string) error {
 	if imageID.IsLatest() {
 		imageID.Tag = docker.Latest
 	}
+
+	//get the hash and store it in the image record
+	hash, err := client.dc.GetImageHash(uuid)
```

Comment:
Can this be called in the dfs code instead where you add a param for hash in the PushImage function?
 so, PushImage(image, uuid, hash string) error